web/app/themes/gds/app/Providers/WooCommerceServiceProvider.php

```diff
@@ -147,4 +148,19 @@ public function filterFeatures(array $features): array
 
         return $features;
     }
+
+    /* Change price display for Variable Products */
+    public function changeVariablePriceDisplay($price, $productObj)
+    {
+        if ($productObj->get_type() !== 'variable') {
```

Comment:
```suggestion
        if (!($productObj instanceof WC_Product_Variable)) {
```

should give vscode type hinting to know the method call below exists.